ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml

```diff
@@ -283,7 +296,7 @@
         <div>
             <a class="ui red button" href="/moderation/user/@Model.ProfileUser.UserId/wipePlanets">
                 <i class="trash alternate icon"></i>
-                <span>Wipe user's earth decorations</span>
+                <span>Wipe User's Earth Decorations</span>
```

Comment:
Personal nitpick: removing the apostrophe. If you or Slendy believe it's fine the way it is, then it's probably fine.
```suggestion
                <span>Wipe User Earth Decorations</span>
```